src/engine/Drawing/Animation.ts

```diff
@@ -118,7 +118,10 @@ module ex {
             
             this.naturalWidth = images[0] ? images[0].naturalWidth : 0;
             this.naturalHeight = images[0] ? images[0].naturalHeight : 0;
+
+            this.freezeFrame = images.length - 1;
```

Comment:
@jedeen Do we also want to reset this if this.sprites ever changes length? Adding more sprites to an animation would break this, although I can't remember ever doing something like that. It would be an edge case... thoughts?